visualization/app/codeCharta/ui/dialog/dialog.download.component.ts

```diff
@@ -65,6 +67,13 @@ export class DialogDownlodController {
 		})
 	}
 
+	private getAmountOfAttributeTypes(attributeTypes: AttributeTypes) {
+		let sum: number = 0
+		sum += attributeTypes.nodes ? attributeTypes.nodes.length : 0
```

Comment:
attributeTypes.nodes shouldn't be null

Reply:
wel,l not after we merged #549

Reply:
I guess it depends, which branch gets merged first